crates/bevy_tasks/Cargo.toml

```diff
@@ -16,6 +16,8 @@ keywords = ["bevy"]
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+tracing = { version = "0.1", features = ["release_max_level_info"] }
```

Comment:
sadly I cannot use `bevy_log` due to a circular dependency issue...

Reply:
you can depend on `bevy_utils` that re-exports tracing, that's what `bevy_log` is doing

Reply:
I though I tried that already (and ran into a similar issue) but I'll give it another whirl haha